src/context/VerifyContext.js

```diff
@@ -47,8 +47,7 @@ export function VerifyProvider(props) {
   const [currentPage, setCurrentPage] = useState(0);
   const [filter, setFilter] = useState(
     new FilterModel({
-      approved: false,
-      active: true,
+      status: 'unprocessed',
```

Comment:
Best to define this somewhere rather than use a string literal.

Reply:
I called it "captureStatus" instead of "verificationStatus" because that looked too close to "verificationStates" and I wanted to avoid future mistakes.  I can update the naming of similar variables throughout Captures when I work on that section.  Please let me know if there is a better way to go.